packages/shared/src/components/CommentFeed.tsx

```diff
@@ -43,7 +42,7 @@ export default function CommentFeed<T>({
   const queryResult = useInfiniteQuery<CommentFeedData>(
     feedQueryKey,
     ({ pageParam }) =>
-      request(graphqlUrl, query, {
+      gqlClient.request(query, {
```

Comment:
Most change are just `request(url, query, variables)` to `gqlClient.request(query, variables)`